### PR TITLE
Fix ordering for regression queries.

### PR DESCRIPTION
**Pull request by kuba--**
Signed-off-by: kuba-- <kuba@sourced.tech>

Related: https://github.com/src-d/regression-gitbase/issues/7

### CODE REVIEW

File: _testdata/regression.yml [hunk 1]
```diff
@@ -25,59 +25,59 @@
   Statements:
     - SELECT repository_id,contributor_count FROM (SELECT repository_id, COUNT(DISTINCT commit_author_email) AS contributor_count FROM commits GROUP BY repository_id) AS q ORDER BY contributor_count DESC LIMIT 10
 -
-  ID: 'query06'
+  ID: 'query05'
```
Comment (jfontan) [inline _testdata/regression.yml]: Delete all queries that scan all `files` table as they take too much time to test.

File: _testdata/regression.yml [hunk 2]
```diff
@@ -25,59 +25,59 @@
...
   Name: 'Create pilosa index on language UDF'
   Statements:
     - CREATE INDEX language_idx ON files USING pilosa (language(file_path, blob_content)) WITH (async = false)
     - DROP INDEX language_idx ON files
 -
-  ID: 'query08'
+  ID: 'query06'
```
Comment (jfontan) [inline _testdata/regression.yml]: Delete

File: _testdata/regression.yml [hunk 3]
```diff
@@ -25,59 +25,59 @@
...
     - DROP INDEX language_idx ON files
 -
-  ID: 'query09'
+  ID: 'query07'
   Name: 'Query all files from HEAD'
   Statements:
     - SELECT cf.file_path, f.blob_content FROM ref_commits r NATURAL JOIN commit_files cf NATURAL JOIN files f WHERE r.ref_name = 'HEAD' AND r.history_index = 0
 -
-  ID: 'query11'
+  ID: 'query08'
```
Comment (jfontan) [inline _testdata/regression.yml]: Delete

File: _testdata/regression.yml [hunk 4]
```diff
@@ -25,59 +25,59 @@
...
     - SELECT * FROM (SELECT commit_author_email as author, repository_id as id, count(*) as num_commits FROM commits GROUP BY commit_author_email, repository_id) AS t ORDER BY num_commits DESC
 -
-  ID: 'query14'
+  ID: 'query11'
   Name: 'Top committers in all repositories'
   Statements:
     - SELECT * FROM (SELECT commit_author_email as author,count(*) as num_commits FROM commits GROUP BY commit_author_email) t ORDER BY num_commits DESC
 -
-  ID: 'query16'
+  ID: 'query12'
```
Comment (jfontan) [inline _testdata/regression.yml]: Delete

**Comment by kuba--** [comment]
@jfontan - done (removed all FROM files queries).

**Comment by jfontan** [comment]
Thanks @kuba--! Query 5 should also be deleted as it scans all files table creating the repo. The rest LGTM.

**Comment by kuba--** [comment]
@jfontan - query5? I removed all files queries. And right now query5 looks like:
```
  ID: 'query05'
  Name: 'Create pilosa index on language UDF'
  Statements:
    - CREATE INDEX language_idx ON files USING pilosa (language(file_path, blob_content)) WITH (async = false)
    - DROP INDEX language_idx ON files
```

**Comment by jfontan** [comment]
Yes, that one. Creating that index means scanning the whole `files` table and takes hours with the current set of repositories.

**Comment by kuba--** [comment]
I've rename docs/assets/gitbase-db-diagram.png → docs/assets/gitbase-schema.png to trigger gitbook refresh.

**Pull request merged**